package.json

```diff
@@ -264,6 +264,7 @@
         "@hapi/address": "^2.0.0",
         "axios": "^0.19.0",
         "classnames": "^2.2.5",
+        "color": "^3.0.0",
```

Comment:
This is another 7.5KB of gzipped code for a single usage. Can we find a different way, please?

Reply:
its already included in our apps, its used for theming

Reply:
Sure, but it's not currently used in Elements and will increase bundle size for platform customers. It's also used in a single place. Is there no other alternative?

Reply:
Have to keep library consistent otherwise there will be 2 libraries.
I haven't seen other ones, maybe tinycolor

Reply:
https://www.npmtrends.com/chroma-js-vs-color-vs-color-js-vs-tinycolor-vs-tinycolor2